src/components/ha-attribute-value.ts

```diff
@@ -44,7 +42,7 @@ class HaAttributeValue extends LitElement {
                 ${attributeValue}
               </a>
             `;
-        } catch (_) {
+        } catch {
```

Comment:
Can we use this? Does Babel transpile this for browsers that dont support it (and should we)?